### PR TITLE
chore: Update Android Gradle plugin installation

### PR DESCRIPTION
**Pull request by cthomas-newrelic**
We don't support Gradle Plugin Portal, so this adds instructions for using the new plugin spec which assumes plugins all live in the portal.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

What problems does this PR solve?
The change provides information to workaround our missing support for a Gradle Plugin Portal community artifact.

Add any context that will help us review your changes such as testing notes, links to related docs, screenshots, etc.
This addition will be temproary until we publish to GPP ( should be in the next month or two)

If your issue relates to an existing GitHub issue, please link to it.
Mobile agents [Jira ticket here](https://new-relic.atlassian.net/browse/NR-145885)

### CODE REVIEW

**Comment by cthomas-newrelic** [comment]
Hi @akristen - do you know why the Gatsby build is failing (timing-out)? It woudl be helpful to see the changes in context.

**Comment by akristen** [comment]
@cthomas-newrelic I'm looking into the build issue for you. Will circle back with updates / if I need to loop Eng in for ya

File: src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx [hunk 1]
```diff
@@ -60,16 +60,73 @@ These procedures to configure your Android app with Gradle and Android Studio al
...
         ```
         </Collapser>
+
+       <Collapser
+          id="app-level"
+          title="Using the Gradle DSL with Gradle Plugin Portal"
+        >
+
+        <Callout variant="important">
+          By default, the plugins {} DSL resolves plugins from the public <a href="https://plugins.gradle.org/">Gradle Plugin Portal</a>Gradle Plugin Portal</a>.
```
Comment (akristen) [inline src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-android-gradle-android-studio.mdx:70]:
```suggestion
          By default, the plugins {} DSL resolves plugins from the public <a href="https://plugins.gradle.org/">Gradle Plugin Portal</a>Gradle Plugin Portal.
```

**Comment by akristen** [comment]
Closing this PR as it's being picked up in a separate PR: https://github.com/newrelic/docs-website/pull/15257

**Pull request closed without merging**